src/revpi_device_info/cli.py

```diff
@@ -54,46 +54,53 @@ def output_text(device_info: RevPiDeviceInfo, line_length: int = 60):
     print(f"Format Version:\t{device_info.format_version}")
 
 
-def main() -> None:
+def main() -> int:
     parser = argparse.ArgumentParser(
-        description="Human readable RevPi Device Info")
+        description="Human readable RevPi Device Info",
```

Comment:
This is some kind of white space change. This is from your IDE? seperate commit?

Reply:
This has been changed on purpose. This results in a cleaner diff when adding parameters and improves code readability.

Reply:
Please do these changes in an extra commit. The best would be as first commit.

Reply:
I would have done that for a larger project. This project is very new and tiny, the change file has less than 100 lines of source code. I think it's okay the way it is in this case.

Reply:
So splitting the commits should not be much of a burden.